blocks/init/src/blocks/components/copyright/copyright-style.scss

```diff
@@ -0,0 +1,3 @@
+// .copyright {
```

Comment:
Why commented?

Reply:
No style is needed by default. But can be added if you need it. 🤷‍♂

Reply:
I think we do / did the same for other things / components.